src/emqtt_cli.erl

```diff
@@ -273,6 +281,14 @@ parse_cmd_opts([{repeat, Repeat} | Opts], Acc) ->
     parse_cmd_opts(Opts, [{repeat, Repeat} | Acc]);
 parse_cmd_opts([{repeat_delay, RepeatDelay} | Opts], Acc) ->
     parse_cmd_opts(Opts, [{repeat_delay, RepeatDelay} | Acc]);
+parse_cmd_opts([{print, WhatToPrint} | Opts], Acc) ->
+    parse_cmd_opts(Opts, [{print, WhatToPrint} | Acc]);
+parse_cmd_opts([{verify, IsVerify} | Opts], Acc) ->
+    V = case IsVerify of
+            true -> verify_peer;
+            false -> verify_none
+        end,
+    parse_cmd_opts(Opts, maybe_append(ssl_opts, {verify, V}, Acc));
 parse_cmd_opts([_ | Opts], Acc) ->
```

Comment:
Q: don't we need to add a clause for `sni` for it to take effect?

Reply:
Indeed, I missed that part while backporting.